Made some minor contributions (#1157)
MicrosoftDocs/quantum-docs-pr: articles/overview/understanding-quantum-computing.md

# Understanding quantum computing

> Quantum computing uses the principles of quantum mechanics to process information. Because of this, quantum computing requires a different approach than classical computing. One example of this difference is the processor used in quantum computers. Where classical computers use familiar silicon-based chips, quantum computers use quantum systems(controlling the unique properties and behaviours of quanta) such as atoms, ions, photons, or electrons. Classical computers use binary logic based highs and lows i.e. 1 and 0 respectively. While Quantum computers use their quantum properties to represent bits that can be prepared in different quantum superpositions of 1 and 0.  

Reviewer: ```suggestion
Quantum computing uses the principles of quantum mechanics to process information. Because of this, quantum computing requires a different approach than classical computing. One example of this difference is the processor used in quantum computers. Where classical computers use familiar silicon-based chips, quantum computers use quantum systems (systems in which we can control the unique properties and behaviors of quantum mechanics) such as atoms, ions, photons, or electrons. While Classical computers use binary logic based highs and lows i.e. 1 and 0 respectively, Quantum computers use their quantum properties to represent bits that can be prepared in different quantum superpositions of 1 and 0.  
```

> Quantum computing uses the principles of quantum mechanics to process information. Because of this, quantum computing requires a different approach than classical computing. One example of this difference is the processor used in quantum computers. Where classical computers use familiar silicon-based chips, quantum computers use quantum systems (systems in which we can control the unique properties and behaviors of quantum mechanics) such as atoms, ions, photons, or electrons. While Classical computers use binary logic based highs and lows i.e. 1 and 0 respectively, Quantum computers use their quantum properties to represent bits that can be prepared in different quantum superpositions of 1 and 0.  

Reviewer 2: ```suggestion
Quantum computing uses the principles of quantum mechanics to process information. Because of this, quantum computing requires a different approach than classical computing. One example of this difference is the processor used in quantum computers. Where classical computers use familiar silicon-based chips, quantum computers use quantum systems (systems in which we can control the unique properties and behaviors of quantum mechanics) such as atoms, ions, photons, or electrons. While classical computers use binary logic-based highs and lows i.e. 1 and 0 respectively, quantum computers use their quantum properties to represent bits that can be prepared in different quantum superpositions of 1 and 0.  
```
